src/server/db/uploads.js

```diff
@@ -41,11 +41,10 @@ function getUpload (id, trns = knex) {
 }
 
 function validForReportingPeriod (period_id, trns = knex) {
```

Comment:
What do you think about renaming this to `exportedForReportingPeriod`?  I was a little confused at first because I thought this function returned "all" valid uploads for the reporting period.

Reply:
this would be a merge conflict with a PR you have open (auditing) so i'll do it as a follow-up.